test/non_smart_contracts_with_balance_23_03.csv

```diff
@@ -0,0 +1,98 @@
+ADDRESS,BALANCE
```

Comment:
Noticing an inconsistency with the holder count. Seems to be one missing according to [PolygonScan](https://polygonscan.com/token/0x11d9efDf4Ab4A3bfabf5C7089F56AA4F059AA14C). 110 on PolygonScan, 109 in this Snapshot and last transaction was 4 days ago (before snapshot). Might be good to double check the balances between these two files adds up to the total supply.

Reply:
Interesting, I did not think of using this csv feature, it's a great sanity check! 

The original motivation was to automate detecting the smart contract accounts, which will aid us in the migration. 
I will review the diff and see what's causing the discrepancy.

Reply:
Looked into this, looks like polyscan does not produce the CSV on demand but produces it periodically and caches it 

for example this address 0x5299c7d2b73b6a96231081dabfd54dfcc84fedeb shows up in the export csv 

[export-tokenholders-for-contract-0x11d9efDf4Ab4A3bfabf5C7089F56AA4F059AA14C.csv](https://github.com/BreadchainCoop/bread-token/files/14831724/export-tokenholders-for-contract-0x11d9efDf4Ab4A3bfabf5C7089F56AA4F059AA14C.csv)

but calling balanceOf on this address [here](https://polygonscan.com/token/0x11d9efDf4Ab4A3bfabf5C7089F56AA4F059AA14C#code) returns 0. 

However looking at his profile [here](https://polygonscan.com/address/0x5299c7d2b73b6a96231081dabfd54dfcc84fedeb) shows he does have a bread balance, when he in fact does not. 

I'm deferring to my script because of the polyscan inconsistency, and because I have a checksum at the end and the total supply of all accounts lines up with the totalSupply() function in the contract

seems like this ended up being an insanity check :))